css/components/wishlist.css

```diff
@@ -0,0 +1,15 @@
+.wishlist-container {
+  padding: 6rem 1rem 1rem 1rem;
```

Comment:
you can store these spacing values in variables and reuse it so that you can have consistent spacing throughout your application.